src/net.h

```diff
@@ -617,11 +617,27 @@ uint16_t GetListenPort();
 class NetEventsInterface
 {
 public:
-    virtual bool ProcessMessages(CNode* pnode, std::atomic<bool>& interrupt) = 0;
-    virtual bool SendMessages(CNode* pnode) = 0;
+    /** Initialize a peer by adding it to mapNodeState and pushing a message requesting its version */
```

Comment:
This comment about the implementation (`mapNodeState`) doesn't make sense in the interface class. Same for `FinalizeNode()` below.

Reply:
Grr, edited twice, but rebased three times and lost them, I guess. Rebased and edited again.